newfile.txt

```diff
@@ -0,0 +1,3 @@
+checking new file here 
+testing 123
+
```

Comment:
- [ ] [OPTIONAL] You may also consider removing this file as I has nothing to do with the current project